airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -94,17 +107,26 @@ definitions:
         type: string
         enum: [BasicHttpAuthenticator]
       username:
+        title: Username
         description: The username that will be combined with the password, base64 encoded and used to make requests
         type: string
+        examples:
+          - "{{ config['username'] }}"
+          - "{{ config['api_key'] }}"
       password:
+        title: Password
         description: The password that will be combined with the username, base64 encoded and used to make requests
         type: string
         default: ""
+        examples:
+          - "{{ config['password'] }}"
+          - ""
       $parameters:
         type: object
         additionalProperties: true
   BearerAuthenticator:
-    description: Authenticator for requests authenticated with a Bearer token
+    title: Bearer Token Authenticator
```

Comment:
calling it `token` instead of api_key. It's inconsistent with the field name, but that's what it's called https://swagger.io/docs/specification/authentication/bearer-authentication/